### ISSUE TITLE
Validation issue: Name field in Flow

### ISSUE THREAD

**Issue by pooja1425** [glific/glific-frontend#992]
**Describe the bug**
Error validation is not working for duplicate flow name.

**To Reproduce**
1. Create a flow with any name
2. Try to create a flow with same name

**Expected behavior**
It should give us "Name already exists" error message while typing.

**Screenshots**
![Glific_ Two way ](https://user-images.githubusercontent.com/62100957/107472978-e9b22f00-6b95-11eb-9bbc-071acd452dc8.png)

**Comment by abhi1203** [comment]
I agree, Maybe we should just mention in the heading Sorry! the flow name already exists. and no subtext.

**Comment by rathorevaibhav** [comment]
@pooja1425 can you elaborate on the issue here? I can see the error message showing up when I try to save with a duplicate name. Is that something you don't see?

**Comment by pooja1425** [comment]
Previously, the error "Name already exists" was thrown on the input change. But now I think the error comes from the backend and that's why it appears in the pop-up. 
Now we just have to change the format of the error message.
